### DISCUSSION TITLE
editing-channels-and-messages

### DISCUSSION THREAD

**Discussion by giscus** [brainboxdotcc/dpp-comments#43] [General]
# editing-channels-and-messages

A lightweight C++ Discord API library supporting the entire Discord API, including Slash Commands, Voice/Audio, Sharding, Clustering and more!

https://dpp.dev/editing-channels-and-messages.html

**Comment by AndraxFlow** [comment]
How can i delete a channel i created?

**Reply by Henonicks** [reply]
`dpp::cluster::channel_delete`
Hi, sorry, didn't know anyone would comment here. If there was a way, I'd subscribe.
